### PR TITLE
add chroot-mode, so it works without qemu/kernel

### PR DESCRIPTION
**Pull request by bittorf**
Like proposed in IRC from user 'mihi'.
This makes it possible to testrun it without a kernel/qemu.

### CODE REVIEW

File: rootfs.sh [hunk 1]
```diff
@@ -114,13 +114,18 @@ cd tmp
 find . | cpio -H newc -o | gzip > initramfs.igz
 
 # Run
-${QEMU_CMD:-qemu-system-x86_64} -enable-kvm \
-    -m "${RAM:-8G}" \
-    -nographic \
-    -no-reboot \
-    -kernel ../../kernel -initrd initramfs.igz -append console=ttyS0
+if [ "$QEMU_CMD" = 'chroot' ]; then
+	PATH="/after/bin:$PATH" sudo chroot . /init
```
Comment (stikonas) [inline rootfs.sh:118]: maybe swap `sudo` and `PATH`
```suggestion
	sudo PATH="/after/bin:$PATH" chroot . /init
```

File: rootfs.sh [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-RAM=$2
+QEMU_CMD="${1:-qemu-system-x86_64}"	# or 'chroot'
+QEMU_RAM="${2:-8G}"
 
 pushd sysa
 
 # SYSTEM A
 
-# Setup tmp 
+echo "trying to mount ramdisk/tmpfs: size 8G"
```
Comment (fosslinux) [inline rootfs.sh:11]: Don't think this is necessary...

File: rootfs.sh [hunk 3]
```diff
@@ -114,13 +114,18 @@ cd tmp
...
 # Run
-${QEMU_CMD:-qemu-system-x86_64} -enable-kvm \
-    -m "${RAM:-8G}" \
-    -nographic \
-    -no-reboot \
-    -kernel ../../kernel -initrd initramfs.igz -append console=ttyS0
+if [ "$QEMU_CMD" = 'chroot' ]; then
+	PATH="/after/bin:$PATH" sudo chroot . /init
+else
+	$QEMU_CMD -enable-kvm \
```
Comment (fosslinux) [inline rootfs.sh:120]: Please change `$QEMU_CMD` to `"${QEMU_CMD}"`.

**Review by fosslinux — CHANGES_REQUESTED** [review]

File: rootfs.sh (hunk 3 above)
Comment (stikonas) [reply rootfs.sh:120]: Is this necessary? `shellcheck` doesn't complain about this and I think there is no risk of escaping here.

**Pull request closed without merging**